src/components/MarketingBanner/MarketingBanner.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer"
+import CircularProgress from "@mui/material/CircularProgress"
+import {
+  Locales,
+  getAssetUrl,
+  useGetContentfulEntry,
+} from "../../hooks/contenful"
+import {
+  IBannerFields,
+  LowercasedAlignment,
+  MarketingBannerProps,
+} from "./MarketingBanner.types"
+import {
+  BannerContainer,
+  Button,
+  ButtonContainer,
+  Content,
+  LoadingContainer,
+  Logo,
+  Text,
+  Title,
+} from "./MarketingBanner.styled"
+
+export const MarketingBanner: React.FC<MarketingBannerProps> = ({
+  id,
+  environment,
+  token,
+  space,
+  locale = Locales.enUS,
+}) => {
```

Comment:
Same here about extracting the props inside the function so we have consistency throw all the repo

Reply:
Done!